packages/decoder/README.md

```diff
@@ -94,99 +94,6 @@ Note that decoding of state variables is only available in full mode; attempting
 to decode state variables will result in an exception if full mode is not
 possible.
 
-### Output format: `Type`s and `Result`s
```

Comment:
Hold on, all this is just gone from the README??  I'd suggest at least including a link.  Like, leave the section in, but rename it and just make the content a link.  I think this is too important to not appear in the README, even if it's no longer going to appear directly.

Reply:
Heh ok fine I was being lazy and saving that for another PR.

Will update!